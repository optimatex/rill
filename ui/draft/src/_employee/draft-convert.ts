export { convertToHTML, convertFromHTML } from 'draft-convert';
