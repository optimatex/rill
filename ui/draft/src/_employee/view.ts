export {
  React,
  useState,
  useCallback,
  css,
  styled,
  createGlobalStyle,
} from '@rill/view';
