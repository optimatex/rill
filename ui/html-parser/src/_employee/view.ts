export { React, RC } from '@rill/view';
