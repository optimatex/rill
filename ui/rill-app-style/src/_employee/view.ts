export { createGlobalStyle, css } from '@rill/view';
