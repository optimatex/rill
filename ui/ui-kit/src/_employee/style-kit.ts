export {
  // appLayout
  appLayoutCss,
  screenCss,
  sidebarColumnCss,
  flatColumnCss,
  mainColumnCss,
  // layout
  listCss,
  // Brand
  logCss,
  dualLogoCss,
  // Controller
  trivialAppHeaderCss,
  // _menu
  menuWrapCss,
  menuItemCss,
  // Image
  imageCss,
  imagesListCss,
  imagesGridCss,
  imagesLayoutItemCss,
  imageCropperCss,
  // Tags
  tag,
  tagsList,
  // Ui-primal
  // _grid
  gridRowContainerCss,
  gridrillRowContainerCss,
  gridColumnCss,
  // _card
  cardContainerCss,
  postContainerCss,
  postTitleCss,
  panelCss,
  // _buttons
  buttonCss,
  buttonActiveCss,
  buticonCss,
  // _form
  formContainerCss,
  formFieldFlatCss,
  formFieldColumnCss,
  formFieldErrorCss,
  formLabelTextCss,
  inputCss,
  toggleContainerCss,
  toggleContainerActiveCss,
  toggleRoundCss,
  toggleRoundActiveCss,
  // _article
  paragraphCss,
  nameCss,
  richContentWrapCss,
  // _table
  tableContainerCss,
  tableHeadCss,
  tableRowCss,
  tableColumnCss,
  // Unknown
  // _object
  stringValueCss,
  numberValueCss,
  booleanValueCss,
  emptyValueCss,
  arrayWrapCss,
  arrayItemCss,
  objectKeyCss,
  objectWrapCss,
  objectPairCss,
  // Unsorted
  lightIndicatorCss,
  popoverCss,
  pipeCss,
  hintableLogCss,
  fixedPanelCss,
} from '@rill/style-kit';
