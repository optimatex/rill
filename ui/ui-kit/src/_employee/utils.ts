export {
  toPairs,
  find,
  has,
  path,
  prop,
  curry,
  assoc,
  lensProp,
  view,
  set,
} from '@rill/utils';
