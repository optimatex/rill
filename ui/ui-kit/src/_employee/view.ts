export {
  React,
  useState,
  useCallback,
  // styled-components
  styled,
  // Lens
  // getTestId,
  // Types
  FC,
  RC,
  ReactNode,
  LegacyRef,
  // PropTypes
  IContainerProps,
  // Event types
  FormEvent,
  MouseEvent,
} from '@rill/view';
