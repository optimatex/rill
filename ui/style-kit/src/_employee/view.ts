import { css as originalCss } from '@rill/view';

export const css = originalCss;
