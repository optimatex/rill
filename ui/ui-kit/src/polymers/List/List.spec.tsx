import { React } from '_employee/view';
import { render } from 'test';
import { List } from '.';

describe('List', () => {
  const props = {
    data: ['frodo', 'sam', 'golum'],
    renderItem: (i: string) => <div key={i}>{i}</div>,
    testID: 'Uber Eats Delivery',
  };

  it('should render without errors', () => {
    const { getByTestId } = render(<List {...props} />);
    const el = getByTestId('Uber Eats Delivery');

    expect(el).toMatchSnapshot();
  });
});
