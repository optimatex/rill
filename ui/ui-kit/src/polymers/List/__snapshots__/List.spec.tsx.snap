// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should render without errors 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  list-style-type: none;
  padding: 0;
  margin: 0;
}

.c0 > * {
  margin-bottom: px;
}

.c0 > *:last-child {
  margin-bottom: 0;
}

<ul
  class="c0"
  data-testid="Uber Eats Delivery"
>
  <div>
    frodo
  </div>
  <div>
    sam
  </div>
  <div>
    golum
  </div>
</ul>
`;
